src/components/NcSelectTags/NcSelectTags.vue

```diff
@@ -321,7 +321,7 @@ export default {
 				...propsToForward
 			} = this.$props
 
-			return propsToForward
+			return { ...propsToForward, ...this.$attrs }
```

Comment:
Same as the previous component, or there is another reason to pass `$attrs`?

```suggestion
			return propsToForward
```

Reply:
No particular reason. I just forgot to adjust it.